go.mod

```
@@ -0,0 +1,5 @@
+module github.com/elastic/go-perf
+
+go 1.20
+
+require golang.org/x/sys v0.25.0
```

`v0.26.0` requires 1.18 - so it would be a great match.
```suggestion
require golang.org/x/sys v0.26.0
```